server/superdesk/io/rss.py

```diff
@@ -114,17 +115,21 @@ def _update(self, provider):
             if t_entry_updated <= t_provider_updated:
                 continue
 
-            # If the entry references any images, create a package from it,
-            # otherwise treat it as a simple text item (even if it might
-            # reference other media types, e.g. video clips).
+            item = self._create_item(entry, field_aliases)
+            self.add_timestamps(item)
+
+            # If the entry references any images, create picture items from
+            # them and create a package containing them and the entry itself.
+            # If there are no image references, treat entry as a simple text
+            # item, even if it might reference other media types, e.g. videos.
```

Comment:
Yeah, I had the same thought when reading the specifications, since pictures can essentially be (and in fact, as you pointed out, _are_) treated in the same way as any other media file.
@hlmnrmr should we reconsider this and add support for other media types as well?

@amagdas Are the created item/package dictionaries correct otherwise? Nothing  to change?